dockers/base-cuda/Dockerfile

```diff
@@ -91,10 +91,10 @@ RUN \
     # Disable cache \
     export CUDA_VERSION_MM=$(python -c "print(''.join('$CUDA_VERSION'.split('.')[:2]))") && \
     pip config set global.cache-dir false && \
-    # set particular PyTorch version
-    python ./requirements/pytorch/adjust-versions.py requirements/pytorch/base.txt ${PYTORCH_VERSION} && \
-    python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt ${PYTORCH_VERSION} && \
-    python ./requirements/pytorch/adjust-versions.py requirements/pytorch/examples.txt ${PYTORCH_VERSION} && \
+    # set particular PyTorch version \
+    for fpath in `ls requirements/**/*.txt`; do \
```

Comment:
I suggest doing some filtering by grepping for "torch" in the file

Reply:
does grep work also for Windows? what I found does not look so :(
https://lazyadmin.nl/powershell/powershell-grep-select-string/

Reply:
I don't know :/